tests/materials/material_utility_test.cc

```diff
@@ -0,0 +1,213 @@
+#include <cmath>
+#include <limits>
+#include <memory>
+
+#include "Eigen/Dense"
+#include "catch.hpp"
+
+#include "material_utility.h"
+
+//! \brief Check material_utility class for 3D case
+TEST_CASE("Material utility is checked for 3D case", "[material_utility][3D]") {
```

Comment:
Why specifically is it a 3D case?

Reply:
I guess there is no 3D or 2D in this case, everything is generalized as 3D. I will delete them.